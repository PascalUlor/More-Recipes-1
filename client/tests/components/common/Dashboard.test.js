/* eslint-disable */
import React from 'react';
import toJson from 'enzyme-to-json';
import Dashboard from '../../../components/Dashboard.jsx';


describe('<Dashboard/>', () => {
  it('renders dashboard component without crashing', () => {
    const shallowWrapper = shallow(<Dashboard/>);
    expect(toJson(shallowWrapper)).toMatchSnapshot();
  });
});
