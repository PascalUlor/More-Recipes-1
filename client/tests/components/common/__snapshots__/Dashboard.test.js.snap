// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dashboard/> renders dashboard component without crashing 1`] = `
<div
  className="bg-faded"
>
  <div>
    <Connect(NavBar) />
    <div
      className="main-wrapper text-center"
    >
      <Connect(FlashMessage) />
      <MainContents />
    </div>
  </div>
  <Footer />
</div>
`;
